src/Spectre.libHeatmapDataScaling.Tests/ScalingDataUtilsTest.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+* ScalingDataUtilsTest.cpp
+* Tests ScalingDataUtils
+*
+Copyright 2018 Daniel Babiak
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <gtest/gtest.h>
+#include <gmock/gmock-matchers.h>
+#include "Spectre.libHeatmapDataScaling/ScalingDataUtils.h"
+
+using namespace spectre::visualization;
+
+namespace
+{
+    class ScalingDataUtilsTest : public ::testing::Test
+    {
+    protected:
+        std::vector<double> testIntensities;
+        void SetUp() override
+       {
+           testIntensities = { 3.2, 2.69, 1.23, 1.23 };
+       }
+    };
+
+    TEST_F(ScalingDataUtilsTest, returns_normalized_data)
+    {
+        auto result = normalizeData(testIntensities);
+        EXPECT_THAT(result, testing::ContainerEq(std::vector<double>{ 255.0, 188.98477157360404, 0.0, 0.0 }));
```

Comment:
Isn't putting `188.98477157360404` a little bit overspecification? Can we go for some tolerance? GTest already provides such matchers.